tests/test_ell1h.py

```diff
@@ -12,111 +13,239 @@
-        log.debug("Diffs: %s\nMax: %s" % (diffs, np.max(diffs)))
-        assert np.all(diffs < 4e-8), "J1853 residuals test failed."
-
-    def test_J1853_binary_delay(self):
-        # Calculate delays with PINT
-        pint_binary_delay = self.modelJ1853.binarymodel_delay(self.toasJ1853, None)
-        assert np.all(
-            np.abs(pint_binary_delay.value + self.ltbindelay) < 3e-8
-        ), "J1853 binary delay test failed."
-
-    def test_derivative(self):
-        log = logging.getLogger("TestJ1853.derivative_test")
-        test_params = ["H3", "H4", "STIGMA"]
-        self.modelJ1853.H4.value = 0.0  # For test PBDOT
-        self.modelJ1853.STIGMA.value = 0.0
-        testp = tdu.get_derivative_params(self.modelJ1853)
-        delay = self.modelJ1853.delay(self.toasJ1853)
-        # Change parameter test step
-        testp["H3"] = 6e-1
-        testp["H4"] = 1e-2
-        testp["STIGMA"] = 1e-2
-        for p in test_params:
-            log.debug("Runing derivative for %s", "d_delay_d_" + p)
-            ndf = self.modelJ1853.d_phase_d_param_num(self.toasJ1853, p, testp[p])
-            adf = self.modelJ1853.d_phase_d_param(self.toasJ1853, delay, p)
-            diff = adf - ndf
-            if not np.all(diff.value) == 0.0:
-                mean_der = (adf + ndf) / 2.0
-                relative_diff = np.abs(diff) / np.abs(mean_der)
-                # print "Diff Max is :", np.abs(diff).max()
-                msg = (
-                    "Derivative test failed at d_delay_d_%s with max relative difference %lf"
-                    % (p, np.nanmax(relative_diff).value)
-                )
-                if p in ["EPS1DOT", "EPS1"]:
-                    tol = 0.05
-                else:
-                    tol = 1e-3
-                log.debug(
-                    "derivative relative diff for %s, %lf"
-                    % ("d_delay_d_" + p, np.nanmax(relative_diff).value)
-                )
-                assert np.nanmax(relative_diff) < tol, msg
+simple_par = """
+      PSR    J0613-0200
+      LAMBDA 93.7990065496191  1 0.0000000158550
+      BETA   -25.4071326875232  1 0.0000000369013
+      PMLAMBDA 2.1192  1 0.0174
+      PMBETA -10.3422  1              0.0433
+      PX 0.9074  1              0.1509
+      F0 326.6005670972169810  1  0.0000000000066373
+      F1 -1.022985317101D-15  1  6.219122230955D-20
+      PEPOCH        54890.000000
+      DM         38.778683
+      BINARY ELL1H
+      A1 1.091442190  1  0.000000598
+      PB 1.19851255667964  1 0.00000000001332
+      TASC 54889.991808565  1 0.000000012
+      EPS1 0.0000025554  1 0.0000002783
+      EPS2 0.0000036160  1 0.0000000847
+      H3 2.7507208E-7  1       1.5114416E-7
+      H4 2.0262048E-7  1       1.1276173E-7
+      """
+
+
+@pytest.fixture
+def simple_par_str():
+    return simple_par
+
+
+@pytest.fixture
+def toasJ0613():
+    toas = toa.get_TOAs("J0613-0200_NANOGrav_9yv1.tim", ephem="DE421", planets=False)
+    return toas
+
+
+@pytest.fixture
```

Comment:
If you're confident these will not be modified by any test you can use `@pytest.fixture(scope="module")` to arrange for the TOAs to be loaded only once rather than for every test that uses them.